src/cli/cmds/help.ts

```diff
@@ -14,9 +14,6 @@
  *  limitations under the License.
  */
 
-'use strict';
-
-async function help() {
+export async function help(): Promise<void> {
+  return;
```

Comment:
help is a no-op?

Reply:
For now... Need to add actual help messages (I'll do in another PR. Focusing this one on translating JS => TS)